llvm/include/llvm/Analysis/ScalarEvolution.h

```diff
@@ -1766,6 +1766,7 @@ class ScalarEvolution {
   /// this call will try to use a minimal set of SCEV predicates in order to
   /// return an exact answer.
   ExitLimit computeExitLimit(const Loop *L, BasicBlock *ExitingBlock,
+                             bool ControlsOnlyExit,
```

Comment:
```suggestion
                             bool IsOnlyExit,
```
here and elsewhere. The APIs that use ControlsOnlyExit pass a (controlling) exit condition, using this here doesn't make a lot of sense.

Reply:
Thanks! done.